bindings/python/python/opendal/__base.pyi

```diff
@@ -46,7 +46,7 @@ class _Base:
     @overload
     def __init__(
         self,
-        scheme: Literal["aliyun_drive"],
+        scheme: Literal["aliyun-drive"],
```

Comment:
@Xuanwo aren't some changes required in `dev/src/generate/python.j2` as well. otherwise these will be overwritten by `just generate python`

Reply:
You are right on this! Would you like to have a PR for this? Or I'll create one as follow up.